src/components/panel/panel.scss

```diff
@@ -67,12 +67,6 @@
   @apply hidden;
 }
 
-:host([loading]) .container,
```

Comment:
Did you test this to make sure that the scrim is showing correctly and with scrolling elements below?

Reply:
Yes, but will test again to make sure.

Reply:
Local non-exhaustive testing looks good. 🎉